util/Migrator/DbMigrator.cs

```diff
@@ -14,13 +14,15 @@ public class DbMigrator
     private readonly string _connectionString;
     private readonly ILogger<DbMigrator> _logger;
     private readonly bool _skipDatabasePreparation;
-
+    private readonly bool _noTransactionMigration;
```

Comment:
⛏️ Newline underneath this to separate from the constructor.

Reply:
Added